disnake/http.py

```diff
@@ -101,6 +102,16 @@ def _workaround_set_api_version(version: Literal[9, 10]) -> None:
     Route.BASE = f"https://discord.com/api/v{_API_VERSION}"
 
 
+def get_user_agent() -> str:
```

Comment:
Rename to `_get_user_agent`? `USER_AGENT` is the attribute that should be used in other modules